package.json

```diff
@@ -14,8 +14,8 @@
     "eslint": "npm run lint",
     "lint": "eslint .",
     "servertest": "npm run build && better-npm-run servertest",
-    "start:disco": "better-npm-run start:disco",
-    "start:search": "better-npm-run start:search",
+    "start:disco": "npm run version-check && NODE_PATH='./:./src' NODE_APP_INSTANCE=disco node bin/server.js",
+    "start:search": "npm run version-check && NODE_PATH='./:./src' NODE_APP_INSTANCE=search node bin/server.js",
```

Comment:
Nice, i was begginning to get to the point where I was thinking we should kill all the app specific start up stuff in favour of just using `NODE_APP_INSTANCE` (aside from maybe the dev things for convenience).